README.md

```diff
@@ -32,44 +30,440 @@ The sync service must first be initialized using the `syncService.init()`. Gener
 });
 ```
 
-Once initialized the syncService can manage multiple `dataset` using the following function:
+##### Managing Datasets
+
+Once initialized the syncService can manage multiple `datasets` using the following function:
 
 ```javascript
 
+var config = {
+  ...
+  datasetId: "workorders"
+  ...
+};
+
 syncService.manage(config.datasetId, null, queryParams);
 
 ```
 
-Checkout a full example [here](https://github.com/feedhenry-staff/wfm-workorder/blob/master/lib/angular/sync-service.js)
 
+### Dataset Topic Subscriptions
 
-#### Topics
+Each `dataset` subscribes to the following topics. For a `dataset` with an ID `datasetid`, the following topics are published:
 
-The following topics are published by the client side of the application:
 
-##### Data Set Topics
+#### Topic Unique Identifiers
 
-The following topics are published by the module for each `dataset`. If the 
+Each of the topics takes an `object` as a parameter. The `topicUid` parameter is an *optional* parameter used to allow the unique identifier to be appended to the `done` and `error` topics published.
 
-###### $fh.sync Client Notifications
+This allows for the scenario where a developer wishes to limit the response of the topic.
+
+E.g
+
+```javascript
+
+//Generate a random number / string
+var topicUid = Math.floor(Math.random() * 100000000);
```

Comment:
[Question] I saw we use timestamp for publishing events. Can we use timestamp or have some helper inside mediator to denote that we want to deliver this to single subscriber.

I think that if we want to listen to specific even we should use pipes and queues. Publish subscribe model is best when having more than one receiver. Maybe we can add to mediator option for point to point communication.  WDYT?